WordPress/Classes/ViewRelated/Comments/EditCommentSingleLineCell.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+class EditCommentSingleLineCell: UITableViewCell, NibReusable {
```

Comment:
Is there anything about the new cell that specific for comments or could it be generic enough to use in other situations? If generic we might consider dropping "Comment" from the name.

Reply:
I thought about that, but I'm not quite sure yet. I'm not finished adding logic to this cell. If it can be generic in the end, I'll change it.